core/src/Pos/Core/Configuration/Core.hs

```diff
@@ -41,13 +41,15 @@ data GenesisConfiguration
       -- | Genesis from a 'GenesisSpec'.
     = GCSpec !GenesisSpec
       -- | 'GenesisData' is stored in a file.
-    | GCSrc { gcsFile :: !FilePath
-            -- ^ Path to file where 'GenesisData' is stored. Must be
-            -- in JSON, not necessary canonical.
-            , gcsHash :: !(Hash Raw)
-            -- ^ Hash of canonically encoded 'GenesisData'.
-            }
-    deriving (Show)
+    | GCSrc
+             !FilePath
+      -- ^ Path to file where 'GenesisData' is stored. Must be
+      -- in JSON, not necessary canonical.
+             !(Hash Raw)
+      -- ^ Hash of canonically encoded 'GenesisData'.
+
+
+    deriving (Eq, Show, Generic)
```

Comment:
How come `deriving` is so far separated from the above? I feel like the two line-breaks are unnecessary here.